driver/src/test/java/org/neo4j/driver/v1/tck/CypherComplianceSteps.java

```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright (c) 2002-2016 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.neo4j.driver.v1.tck;
+
+import cucumber.api.DataTable;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.neo4j.driver.v1.ResultCursor;
+import org.neo4j.driver.v1.Value;
+import org.neo4j.driver.v1.tck.tck.util.runners.CypherStatementRunner;
+import org.neo4j.driver.v1.tck.tck.util.runners.MappedParametersRunner;
+import org.neo4j.driver.v1.tck.tck.util.runners.StringRunner;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.neo4j.driver.v1.tck.DriverComplianceIT.session;
+import static org.neo4j.driver.v1.tck.Environment.runners;
+import static org.neo4j.driver.v1.tck.tck.util.ResultParser.getMapFromString;
+import static org.neo4j.driver.v1.tck.tck.util.ResultParser.getParametersFromListOfKeysAndValues;
+import static org.neo4j.driver.v1.tck.tck.util.ResultParser.parseExpected;
+import static org.neo4j.driver.v1.tck.tck.util.ResultParser.parseGiven;
+
+
+public class CypherComplianceSteps
+{
+    @Given( "^init: (.*);$" )
+    public void init_( String statement ) throws Throwable
+    {
+        session.run( statement );
+    }
+
+    @When( "^running: (.*);$" )
+    public void running_( String statement ) throws Throwable
+    {
+        runners.add( new StringRunner( statement ).runCypherStatement() );
+    }
+
+
+    @When( "^running parametrized: (.*);$" )
```

Comment:
Its steps because it is a step-file. But I guess maybe CypherComplianceKitSteps would be better?